File: Problem_09/problem_9.erl
-module(problem_9).
-compile(export_all).

problem_9()->
    lists:nth(1, 
	      [ {1000 - B - C, B, C} || C <- lists:seq(1, 1000), 
					B <- lists:seq(1, C), 
					(1000 - B - C) * (1000 - B - C) + B * B == C * C]).

start()->
    io:format("result: ~p~n", [problem_9()]).

File: Problem_09/problem_9.js
function problem_9() {
    for (var c = 1; c <= 1000; c++) {    
	for (var b = 1; b <= c; b ++) {
	    var a = 1000 - b - c;
	    if (a*a + b*b == c*c) {
		return a + "x" + b + "x" + c;
	    }
	}
    }
}

document.write('<b>result: </b>' + problem_9());